src/decisionengine/framework/engine/Workers.py

```diff
@@ -98,13 +99,13 @@ def run(self):
 
         logconf["loggers"].update(
             {
-                f"{myname}": {
+                CHANNELLOGGERNAME: {
                     "handlers": [f"{myname}", "file_structlog_debug"],
```

Comment:
If you have a str variable you can use it directly (`myname` instead of `f"{myname}"`)

Reply:
Right.